### PR TITLE
Add SQL IT against security enabled cluster

### PR DESCRIPTION
**Pull request by weicongs-amazon**
Does this PR include tests?

692 tests completed, 1 failed, 42 skipped
https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/737098855?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/test-build-deb.yml [hunk 1]
```diff
@@ -365,6 +365,56 @@ jobs:
...
+          sudo apt-get -y update
+          sudo apt install -y opendistroforelasticsearch
+          
+          sudo chmod 777 /etc/elasticsearch/elasticsearch.yml
+          sudo sed -i /^node.max_local_storage_nodes/d /etc/elasticsearch/elasticsearch.yml
+          sudo /etc/init.d/elasticsearch start
+
+          sleep 30
+          cd sql
+          ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Duser=admin -Dpassword=admin
```
Comment (gaiksaya) [inline .github/workflows/test-build-deb.yml:416]: Do we need to add "-Dhttps=true"? Saw it in the quip you shared before

File: .github/workflows/test-build-deb.yml [hunk 2]
```diff
@@ -365,6 +365,56 @@ jobs:
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Run Debian Staging Distrubution
+        run: |
+          cd elasticsearch/linux_distributions
+          es_version=`../bin/version-info --es`
+          echo $es_version
+          cd ../../..
```
Comment (peterzhuamazon) [inline .github/workflows/test-build-deb.yml:392]: You can change to this one now following #227 
``es_version=`./bin/version-info` --es``
``echo $es_version``

File: .github/workflows/test-build-deb.yml (hunk 1 above)
Comment (weicongs-amazon) [reply .github/workflows/test-build-deb.yml]: good catch. missed it when I copied from another branch

File: .github/workflows/test-build-deb.yml (hunk 2 above)
Comment (weicongs-amazon) [reply .github/workflows/test-build-deb.yml]: update it, thanks

**Pull request merged**